src/window/win.cc

```diff
@@ -887,9 +887,18 @@ namespace SSC {
 
                   EventRegistrationToken tokenNewWindow;
 
+                  // TODO(trevnorris): This callback is only called when window.open() is called,
+                  // but for some reason the callback is never being called from the socket-api
+                  // tests. Because of this we can't store requested_event_args, which means that
+                  // getSize() can't properly retrieve the width and height of the window.
                   webview->add_NewWindowRequested(
                     Microsoft::WRL::Callback<ICoreWebView2NewWindowRequestedEventHandler>(
                       [&](ICoreWebView2* wv, ICoreWebView2NewWindowRequestedEventArgs* e) {
+                        Window* w = reinterpret_cast<Window*>(GetWindowLongPtr((HWND)window, GWLP_USERDATA));
+                        if (w->requested_event_args == nullptr) {
+                          abort();
```

Comment:
Should we print a warning for this?

Reply:
Adding the `abort()` was for testing. `requested_event_args` should be `nullptr` at this point, and I wanted the program to abort so I could collect some info in VS. Problem is this callback is never called. So I could never get it to abort.